mmdet/models/backbones/xception.py

```diff
@@ -0,0 +1,317 @@
+                                                       self.conv1.out_channels)
+        self.relu = nn.ReLU(inplace=True)
+
+        self.conv2 = build_conv_layer(
+            self.conv_cfg,
+            self.conv1.out_channels,
+            self.conv_sizes[1],
+            3,
+            bias=False)
+        if self.bn:
+            self.bn2_name, self.bn2 = build_norm_layer(self.norm_cfg,
+                                                       self.conv2.out_channels)
+        # do relu here
+
+        self.block1 = Block(
+            self.conv_cfg,
+            self.conv2.out_channels,
+            self.conv_sizes[2],
+            2,
+            stride=2,
+            start_with_relu=False,
+            grow_first=True,
+            norm_cfg=self.norm_cfg)
+        self.block2 = Block(
+            self.conv_cfg,
+            self.block1.out_channels,
+            self.conv_sizes[3],
+            2,
+            stride=2,
+            start_with_relu=True,
+            grow_first=True,
+            norm_cfg=self.norm_cfg)
+        self.block3 = Block(
+            self.conv_cfg,
+            self.block2.out_channels,
+            self.conv_sizes[4],
+            2,
+            stride=2,
+            start_with_relu=True,
+            grow_first=True,
+            norm_cfg=self.norm_cfg)
+
+        self.center = nn.Sequential(
+            OrderedDict([('block' + i,
+                          Block(
+                              self.conv_cfg,
+                              self.conv_sizes[4],
+                              self.conv_sizes[4],
+                              3,
+                              stride=1,
+                              start_with_relu=True,
+                              grow_first=True,
+                              norm_cfg=self.norm_cfg))
+                         for i in range(self.depth)]))
+
+        self.last_block = Block(
+            self.conv_cfg,
+            self.conv_sizes[4],
+            self.conv_sizes[5],
+            2,
+            stride=2,
+            start_with_relu=True,
+            grow_first=False,
+            norm_cfg=self.norm_cfg)
+
+        self.conv3 = SeparableConv(self.conv_cfg, self.last_block.out_channels,
+                                   self.conv_sizes[6], 3, 1, 1)
+        if self.bn:
+            self.bn3_name, self.bn3 = build_norm_layer(self.norm_cfg,
+                                                       self.conv3.out_channels)
+
+        # do relu here
+        self.conv4 = SeparableConv(self.conv_cfg, self.conv3.out_channels,
+                                   self.conv_sizes[7], 3, 1, 1)
+        if self.bn:
+            self.bn4_name, self.bn4 = build_norm_layer(self.norm_cfg,
+                                                       self.conv4.out_channels)
+
+        # self.fc = nn.Linear(2048, num_classes)
+
+    def _init_weights(self, ):
+        for module in self.modules():
+            if isinstance(module, nn.Conv2d):
```

Comment:
Use the implemented initialization methods (constant_init, kaiming_init) implemented in MMCV, unless the initialization is special and not implemented in MMCV. If so, please also add comments in the code as a reminder.